package-lock.json

```diff
@@ -0,0 +1,4424 @@
+{
```

Comment:
guess it made sense in this case to change the yarn.lock with new contentful library? With this one I had some doubts because I can't  do yarn install --frozen-lockfile even removing direction in a previous step. So made yarn install but wanted to checkout with you this.